feature_layers/create-mobile-geodatabase/src/main/resources/create_mobile_geodatabase/main.fxml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright 2022 Esri.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License"); you may not
+  ~ use this file except in compliance with the License. You may obtain a copy of
+  ~ the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  ~ WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  ~ License for the specific language governing permissions and limitations under
+  ~ the License.
+  -->
+<?import javafx.scene.layout.StackPane?>
+<?import javafx.scene.layout.VBox?>
+<?import javafx.scene.control.Button?>
+<?import javafx.scene.control.Label?>
+<?import javafx.geometry.Insets?>
+<?import javafx.scene.layout.Background?>
+<?import javafx.scene.layout.BackgroundFill?>
+<?import javafx.scene.paint.Color?>
+
+<?import com.esri.arcgisruntime.mapping.view.MapView?>
+
+<StackPane xmlns="http://javafx.com/javafx"
+           xmlns:fx="http://javafx.com/fxml"
+           fx:controller="com.esri.samples.create_mobile_geodatabase.CreateMobileGeodatabaseController"
+           stylesheets="/create_mobile_geodatabase/style.css">
+    <!--SDK MapView-->
+    <MapView fx:id="mapView"/>
+    <!--Controls box-->
+    <VBox StackPane.alignment="TOP_LEFT" spacing="5" maxWidth="180" maxHeight="100" styleClass="panel-region">
+        <padding>
+            <Insets topRightBottomLeft="10"/>
+        </padding>
+        <background>
+            <Background>
+                <fills>
+                    <BackgroundFill>
+                        <fill>
+                            <Color opacity="0.7"/>
+                        </fill>
+                    </BackgroundFill>
+                </fills>
+            </Background>
+        </background>
+        <StackPane.margin><Insets left="10" top="10"/></StackPane.margin>
+        <Label  fx:id="label"
+                wrapText="true"/>
```

Comment:
Although you've added wrap here, because there is a maxHeight set on the VBox, the label cannot wrap - as there is not space. E.g.:
![image](https://user-images.githubusercontent.com/56556822/184854600-a53eebac-b80a-4e24-ac2c-1a0e26e1703e.png)

Rather than over complication I'd just suggest changing the VBox maxWidth to 200 to give plenty of room.
I also propose that you have an initial/default label too so there isn't just a blank space. Something like `Click the button to start.`
![image](https://user-images.githubusercontent.com/56556822/184855991-407806b4-52b4-4a25-bfba-e8285f9d44af.png)